<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="mainFrame">
      <property name="styleSheet">
       <string notr="true">.QFrame {
	background: rgb(245, 245, 245);
	border-radius: 3px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lab_info">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QLabel {
	border:1px solid #b7eb8f;
	border-radius:3px;
	line-height: 140px;
	padding: 5px;
	color: #434343;
	background: #f6ffed;
}</string>
         </property>
         <property name="text">
          <string>一切都准备好！</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QComboBox" name="combo_user">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">.QComboBox {
	border:1px solid #d9d9d9;
	border-radius:3px;
	line-height: 140px;
	padding:5px 20px;
	color: #434343;
}
.QComboBox::drop-down {
	width: 20px;
    border-left: 1px solid #d9d9d9;
}
.QComboBox::down-arrow {
    image: url(:/icon/arrow-down.png)
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_file">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">.QPushButton {
	height: 24px;
    padding: 0px 10px;
    font-size: 14px;
    border-radius: 3px;
    color: rgba(0,0,0,.65);
    background-color: #fff;
    border: 1px solid #d9d9d9;
}

.QPushButton:hover {
	height: 24px;
    padding: 0px 10px;
    font-size: 14px;
    border-radius: 3px;
    color: #40a9ff;
    background-color: #fff;
    border: 1px solid #40a9ff;
}
</string>
           </property>
           <property name="text">
            <string>+自定义路径</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">.QPushButton {
	height: 24px;
    padding: 0px 10px;
    font-size: 14px;
    border-radius: 3px;
    color: rgba(0,0,0,.65);
    background-color: #fff;
    border: 1px solid #d9d9d9;
}

.QPushButton:hover {
	height: 24px;
    padding: 0px 10px;
    font-size: 14px;
    border-radius: 3px;
    color: #40a9ff;
    background-color: #fff;
    border: 1px solid #40a9ff;
}
</string>
           </property>
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <spacer name="horizontalSpacer_15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="check_is_clean">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QCheckBox::indicator::unchecked {   
    image: url(:/icon/复选.svg);
}

QCheckBox::indicator::checked { 
    image: url(:/icon/复选1.svg);
}</string>
           </property>
           <property name="text">
            <string>是否对该账号进行下列内容的清理</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>

        <item>
        <layout class="QHBoxLayout" name="horizontalLayout_100">
         <item>
          <spacer name="horizontalSpacer_100">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
            </font>
           </property>
           <property name="text">
            <string>请勾选你要清理的文件格式：</string>
           </property>
          </widget>
         </item>
        </layout>
        </item>
       
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="check_picdown">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QCheckBox::indicator::unchecked {   
    image: url(:/icon/复选.svg);
}

QCheckBox::indicator::checked { 
    image: url(:/icon/复选1.svg);
}</string>
           </property>
           <property name="text">
            <string>图片</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_14">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="check_files">
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QCheckBox::indicator::unchecked {   
    image: url(:/icon/复选.svg);
}

QCheckBox::indicator::checked { 
    image: url(:/icon/复选1.svg);
}</string>
           </property>
           <property name="text">
            <string>接收的文件（PPT、Doc 等）</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="check_video">
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QCheckBox::indicator::unchecked {   
    image: url(:/icon/复选.svg);
}

QCheckBox::indicator::checked { 
    image: url(:/icon/复选1.svg);
}</string>
           </property>
           <property name="text">
            <string>视频（视频文件和视频的封面图）</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="check_picscache">
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QCheckBox::indicator::unchecked {   
    image: url(:/icon/复选.svg);
}

QCheckBox::indicator::checked { 
    image: url(:/icon/复选1.svg);
}</string>
           </property>
           <property name="text">
            <string>图片缓存（来自小程序、公众号等）</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
            </font>
           </property>
           <property name="text">
            <string>需要保留多久以内的文件：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_gobackdays">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">.QLineEdit {
	border:1px solid #d9d9d9;
	border-radius:3px;
	line-height: 140px;
	padding: 5px;
	color: #434343;
}

.QLineEdit:hover {
	border:1px solid #40a9ff;
	border-radius:3px;
	line-height: 140px;
	padding: 5px;
	color: #434343;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>天数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
            </font>
           </property>
           <property name="text">
            <string>天</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QLabel {
	color: #434343;
}</string>
         </property>
         <property name="text">
          <string>Clean My Wechat V2.1 - 设置页面</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
